pkgs/development/tools/skopeo/default.nix

```diff
@@ -37,8 +37,8 @@ buildGoPackage rec {
   '';
 
   preBuild = ''
-    export CGO_CFLAGS="-I${getDev gpgme}/include -I${getDev libgpgerror}/include -I${getDev devicemapper}/include -I${getDev btrfs-progs}/include"
-    export CGO_LDFLAGS="-L${getLib gpgme}/lib -L${getLib libgpgerror}/lib -L${getLib devicemapper}/lib"
+    export CGO_CFLAGS="-I${getDev gpgme}/include${lib.optionalString stdenv.isLinux " -I${getDev libgpgerror}/include -I${getDev devicemapper}/include -I${getDev btrfs-progs}/include"}"
```

Comment:
Might this make more sense as `CGO_CFLAGS=$CFLAGS`? and same with `LDFLAGS`. I think stdenv sets up the `-I` automatically for us in `CFLAGS` so this would mirror the `buildInputs` conditional logic more naturally.

Reply:
I'll try that.